### ISSUE TITLE
Automated analysis for CFRs

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#3321]
Experiment intake tools may specify feature metrics that Jetstream will analyze. TBD on the feature work being complete.

CFR specific metrics will be available in analysis. Though CFR is not in the Nimbus A/A scope - we want to see CFR for the data analysis sooner. CFR experiments can utilize the automated analysis and provide needed user comprehension/need feedback, before Nimbus intake is ready. Automating CFR analysis is blocked on 1649642.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-50)
┆Issue Number: EXP-50

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

We're closing this because this is being superseded by the work being done in Jetstream
